pymeasure/instruments/common_base.py

```diff
@@ -530,8 +530,46 @@ def fset(self,
         # Add the specified document string to the getter
         fget.__doc__ = docs
 
+        # If dynamic, add (dynamic) to the docstring
```

Comment:
In general, a good heuristic to follow is
> comment "why", not "what" -- the "what" is visible in the code that is commented.

In this case, the comment is just  plain-english rendition of the two line doc snippet, so does not give additional information beyond the code itself.

Reply:
I agree. I felt compelled to put a placeholder comment since I split the dynamic if statement, but the comment doesn't add anything. Fixed in last commit.